Pipfile

```diff
@@ -0,0 +1,16 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+django = "~=2.1"
```

Comment:
@dannylwe please do not fix the django version

Reply:
@ja-odur Hi, I had previously installed Django 2.0 and that was reflected in the Pipfile. I edited the file to change the version to Django 2.1 as is required from the chore. I am not sure how to go about "not fixing" the Django version since it was already fixed at installation?